canonical-web-and-design/ubuntu.com
# Add nodelist foreach polyfill

## Done

- Added a polyfill for `NodeList.forEach`
- Updated tutorials js to use `NodeList.forEach`

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/tutorials
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- In IE 11 click through to a tutorial
- Use the side navigation and check that it works as expected
- Click the feedback face icons on the last step of the tutorial and check that they are replaced by a notification


## Issue / Card

Fixes #6868
